packages/pros/src/adi/ultrasonic.rs

```diff
@@ -0,0 +1,51 @@
+use crate::adi::{
+    AdiError,
+    AdiSlot
+};
+
+use pros_sys::{
+    PROS_ERR,
+    adi_ultrasonic_t, ext_adi
+};
+
+use crate::error::bail_on;
+
+type ext_adi_u8_tuple_t = (u8, u8);
```

Comment:
This naming matches the `pros_sys` naming scheme, but these type defintitions are *probably* unneeded. I would just directly use (u8, u8) where needed rather than aliasing it.